autogluon/utils/forecasting/utils/savers/save_pickle.py

```diff
@@ -0,0 +1,12 @@
+import pickle
+import os
+
+
+def save(path, obj):
```

Comment:
Can you use the existing pickle-save function instead?

https://github.com/awslabs/autogluon/blob/master/core/src/autogluon/core/utils/savers/save_pkl.py#L10